manifests/webhook.pp

```diff
@@ -53,4 +53,8 @@
   contain r10k::webhook::package
   contain r10k::webhook::config
   contain r10k::webhook::service
+
+  Class['r10k::webhook::package']
+  -> Class['r10k::webhook::config']
+  ~> Class['r10k::webhook::service']
```

Comment:
I prefer the dependencies here, but this means we should remove this line? https://github.com/voxpupuli/puppet-r10k/blob/de72a78c1ceb32939a9928f9e5c3801db55ab904/manifests/webhook/config.pp#L10